tensorflow/core/ops/nn_ops.cc

```diff
@@ -3266,6 +3266,7 @@ REGISTER_OP("QuantizedDepthwiseConv2DWithBiasAndRelu")
     .Attr("strides: list(int)")
     .Attr(GetPaddingAttrString())
     .Attr("dilations: list(int) = [1, 1, 1, 1]")
+    .Attr("padding_list: list(int) = []")
```

Comment:
This should be named paddings, to match "dilations" above

Reply:
@alextp thanks for you comments, I have update my code based on it, please help to check, Thanks.

Reply:
@alextp Sorry to revert the change, padding_list  just make consistent with other op like:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc#L2760